modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/util/InlineModelResolver.java

```diff
@@ -454,11 +454,15 @@ public Schema modelFromProperty(Schema object, @SuppressWarnings("unused") Strin
         if (obj != null) {
             example = obj.toString();
         }
-
+        Schema items;
         ArraySchema model = new ArraySchema();
         model.setDescription(description);
         model.setExample(example);
-        model.setItems(object.getAdditionalProperties());
+        if (object.getAdditionalProperties() != null && object.getAdditionalProperties() instanceof Schema) {
+            items = (Schema)  object.getAdditionalProperties();
```

Comment:
It seems the variable item was created only for assignation, if there's no validation or another it might be a good idea to delete it and make the setItems directly model.setItems((Schema)object.getAdditionalProperties());

Reply:
fixed!